packages/expo-image-picker/README.md

```diff
@@ -47,13 +51,49 @@ This package automatically adds the `CAMERA`, `READ_EXTERNAL_STORAGE`, and `WRIT
 <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
 ```
 
-In `AndroidManifest.xml` add the following `activity` within `application`:
+## Config Plugin
 
-```xml
-<activity
-  android:name="com.canhub.cropper.CropImageActivity"
-  android:theme="@style/Base.Theme.AppCompat">
-</activity>
+> This plugin is applied automatically in EAS Build, only add the config plugin if you want to pass in extra properties.
+
+After installing this npm package, add the [config plugin](https://docs.expo.io/guides/config-plugins/) to the [`plugins`](https://docs.expo.io/versions/latest/config/app/#plugins) array of your `app.json` or `app.config.js`:
+
+```json
+{
+  "expo": {
+    "plugins": ["expo-image-picker"]
+  }
+}
+```
+
+Next, rebuild your app as described in the ["Adding custom native code"](https://docs.expo.io/workflow/customizing/) guide.
+
+### API
+
+The plugin provides props for extra customization. Every time you change the props or plugins, you'll need to rebuild (and `prebuild`) the native app. If no extra properties are added, defaults will be used.
+
+- `photosPermission` (_string | false_): Sets the iOS `NSPhotoLibraryUsageDescription` permission message to the `Info.plist`. Setting `false` will skip adding the permission on iOS and **does not** skip the permission on Android. Defaults to `Allow $(PRODUCT_NAME) to access your photos`.
```

Comment:
Could you please explain why it does not skip it on Android?

Reply:
Because the AndroidManifest changes are applied by merging in values from the library AndroidManifest.xml [here](https://github.com/expo/expo/blob/3ee85c4a2005d3da41669ce3346cd5ba27e277e5/packages/expo-image-picker/android/src/main/AndroidManifest.xml#L4-L9). This sits outside of the Expo Config Plugins spec.

Reply:
I've opened a proposal PR to add `android.blockedPermissions` to the `app.json` schema, this can be used to force remove permissions from the final manifest https://github.com/expo/expo-cli/pull/4323